docs/rest_api.md

```diff
@@ -416,6 +416,31 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`POST /node/<node>/dev/<dev>/set_bootdev`
+
+Sets the nodes next boot device persistently
+
+device <dev> can be set to:
```

Comment:
okay, I'll make it a bullet list once we decide what supported boot devices we want (or if we want this list at all).